feg/gateway/docker/build.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
```

Comment:
you can run the following to do some automatic formatting
```
cd $MAGMA_ROOT
./lte/gateway/python/precommit.py --build
./lte/gateway/python/precommit.py --format -p feg/gateway/docker/build.py
```

Reply:
TO run the linter locally just do
```
cd $MAGMA_ROOT
./lte/gateway/python/precommit.py --build
./lte/gateway/python/precommit.py --lint -p feg/gateway/docker/build.py
```

Reply:
awesome. Very useful! didn't know it existed